bin/setup

```diff
@@ -27,11 +27,8 @@ FileUtils.chdir APP_ROOT do
   #   FileUtils.cp "config/database.yml.sample", "config/database.yml"
   # end
 
-  puts "\n== Preparing database =="
-  system! "bin/rails db:prepare"
-
-  puts "\n== Removing old logs and tempfiles =="
-  system! "bin/rails log:clear tmp:clear"
+  puts "\n== Preparing database, removing old logs and tempfiles =="
+  system! "bin/rails db:prepare log:clear tmp:clear"
```

Comment:
I would have expected Spring to skip the boot time but I did observe this slowness as well. I wonder why...